src/e2e/features/createEventWithAllFields.feature

```diff
@@ -0,0 +1,28 @@
+@createEventWithAllFields
+Feature: Event Creation
+
+  Scenario: Verify ECC dashboard page content
+    Given I am on the ECC dashboard page in signed-out state
+    Then I should see the All Events label on the page
+    Then I should see the Search box on the page
+    Then I should see the table headers on the page
+    Then I should see pagination container
+    Then I should see the footer section on the page
+
+    Scenario Outline: Validate and create events with all fields
```

Comment:
scneario name should be in sync with steps, we are first creating and then validating